components/Radio/RadioWrapper.tsx

```diff
@@ -15,6 +15,8 @@ interface RadioWrapperProps {
   readOnly?: boolean
   autoFocus?: boolean
   value?: any
+  groupValue?: string | number
```

Comment:
group value should not be passed to the child

Reply:
Same reason as above, lmk if I'm missing anything! :)